### DISCUSSION TITLE
Colour Mixing

### DISCUSSION THREAD

**Discussion by jac31885** [six7/figma-tokens#885] [Ideas]
Is there potential to include functions such as scss colour and mix in the future? 
Be great to be able to use for creation of 100 - 900 colour values off the one base value.

eg: color.mix({brand.primary}, {color.white}, $weight: 50%)

**Comment by six7** [comment]
Yes! We're thinking of adding something like 'modifiers' that would allow you to mix colors. No ETA yet
